packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -278,25 +353,34 @@ class RecoveryPhoneHandler {
     }
 
     if (success) {
+      this.log.info('account.recoveryPhone.phoneRemoved.success', { uid });
+      await this.glean.twoStepAuthPhoneRemove.success(request);
+
       const account = await this.db.account(uid);
       const { acceptLanguage, geo, ua } = request.app;
 
-      await this.mailer.sendPostRemoveRecoveryPhoneEmail(
-        account.emails,
-        account,
-        {
-          acceptLanguage,
-          timeZone: geo.timeZone,
-          uaBrowser: ua.browser,
-          uaBrowserVersion: ua.browserVersion,
-          uaOS: ua.os,
-          uaOSVersion: ua.osVersion,
-          uaDeviceType: ua.deviceType,
-          uid,
-        }
-      );
-
-      await this.glean.twoStepAuthPhoneRemove.success(request);
+      try {
+        await this.mailer.sendPostRemoveRecoveryPhoneEmail(
+          account.emails,
+          account,
+          {
+            acceptLanguage,
+            timeZone: geo.timeZone,
+            uaBrowser: ua.browser,
+            uaBrowserVersion: ua.browserVersion,
+            uaOS: ua.os,
+            uaOSVersion: ua.osVersion,
+            uaDeviceType: ua.deviceType,
+            uid,
+          }
+        );
+      } catch (error) {
+        // log email send error but don't throw
+        // user should be allowed to proceed
+        this.log.trace('account.recoveryPhone.phoneRemovedNotification.error', {
```

Comment:
I feel like statsd metrics are more useful than logs for monitoring things.

Reply:
I filed https://mozilla-hub.atlassian.net/browse/FXA-11128

We might also want some docs on best practices for logs vs statsd?